### DISCUSSION TITLE
disable or 'ignore' numlock

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20791] [Ideas]
I keep accidentally bumping num lock when playing mania (I play with 456 so I dont know how I hit it, but I somehow do it anyway)
lazer should consider these two identical (idk if its possible to diferentiate between the numpad keys and arrows/text keys) maybe this could be an option in settings.

**Comment by peppy** [comment]
This may help you with an immediate solution: https://superuser.com/questions/688889/permanently-disable-num-lock-in-windows

**Reply by Doksha** [reply]
sadly I use the bottom right of the numpad with numlock disabled for image browsing, and I use it with numlock enabled for number entry, so its not really an option for me

**Reply by peppy** [reply]
You may be able to use microsoft powertoys in that case. It allows setting specific key mapping overrides on a per-app level.

Or you can use sharpkeys / registry changes to map a numlock key somewhere you won't accidentally hit it.
